packages/@lwc/integration-karma/test/custom-elements-registry/index.spec.js

```diff
@@ -3,9 +3,7 @@ const tagAlreadyUsedErrorMessage =
     /(has already been used with this registry|Cannot define multiple custom elements with the same tag name|has already been defined as a custom element|This name is a registered custom element, preventing LWC to upgrade the element)/;
 
 function getCode(src) {
-    return fetch(src)
-        .then((resp) => resp.text())
-        .then((engineCode) => engineCode.replace(/process\.env\.NODE_ENV/g, '"production"'));
+    return fetch(src).then((resp) => resp.text());
```

Comment:
This was causing issues with coverage testing, so I simplified this.

Reply:
Did replacing `process.end.NODE_ENV` with `"production"` result in source-map issues? Not sure I understand what was broken here.

Reply:
IIRC it broke because the string replacement on Istanbul-transformed coverage code was off somehow... Incidentally when I changed some other settings, it worked again, but overall it felt fragile to me, so I prefer this approach.